scripts/dap-ami-bootstrap.sh

```diff
@@ -1,38 +1,42 @@
 #!/bin/bash
 set -e
 
-if ! docker images | grep conjur; then
-  echo "Loading Conjur appliance image into Docker"
-  gzip -df /home/core/conjur-appliance.tar.gz # Unzip the tar.gz into a tar
-  docker load -i /home/core/conjur-appliance.tar
+amazon-linux-extras install -y docker
+systemctl enable docker
+systemctl start docker
+
+if ! docker images | grep appliance; then
+  echo "Loading DAP appliance image into Docker"
+  docker load -i /opt/dap-appliance.tar.gz
 fi
 
 image_id=$(docker images -q)
-container_name='conjur-appliance'
+container_name='dap-appliance'
 
 docker rm -f ${container_name} > /dev/null 2>&1 || true
 
 mkdir -p /var/log/conjur
```

Comment:
What do you think about changing this to `/var/log/dap`?

Reply:
`/var/log/conjur` is also the path inside the container, and is referenced in all of the audit docs.

I have no problem if we change the appliance-internal path to use `dap` as well, but I think they should be the same.

Reply:
I stopped short here for the reason Micah mentioned. I'd love for this to be the start of a wholesale move from "conjur" to "dap", but that's not my call and would require coordination beyond this project.